{{cookiecutter.python_name}}/MANIFEST.in

```diff
@@ -1,3 +1,4 @@
+# WARNING: MANIFEST.in has no effect for hatch backend
```

Comment:
Should we delete this file as the cookiecutter is also for beginner and this may add more confusion?